ts/components/ui/FooterButtons.tsx

```diff
@@ -0,0 +1,95 @@
+import { Button, NativeBase, Text, View } from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+
+export enum FooterButtonsStyle {
+  HALF, // show inline buttons -  1/2 left, 1/2 right    -> [  b1  ] [  b2  ]
+  ONETHIRD, // show inline buttons - 1/3 left, 2/3 right -> [ b1 ] [   b2   ]
+  FULL // show stacked buttons - full size
+}
+
+type OwnProps = {
+  title: string;
+};
+
+type Props = Readonly<{
+  leftButton: NativeBase.Button & OwnProps;
+  rightButton: NativeBase.Button & OwnProps;
+  style: FooterButtonsStyle;
```

Comment:
By convention, the `style` property should contain a style, so this could be confusing, what about using `type` instead?